pkg/validator/summary.go

```diff
@@ -124,10 +124,8 @@ func (c Result) GetSummaryByCategory() CountSummaryByCategory {
 // GetSummary summarizes AuditData
 func (a AuditData) GetSummary() CountSummary {
 	summary := CountSummary{}
-	for _, ctrlResult := range a.Results {
-		if ctrlResult.PodResult != nil {
```

Comment:
any reason this check was removed here? i remember we ran into issues in the past without this check.

Reply:
Yes this is important actually - `res` still has a summary even if it doesn't have a `PodResult`. We're already checking for `PodResult != nil` in the `GetSummary` function so it should be safe.